### ISSUE TITLE
[Feature Request] Mix From any account

### ISSUE THREAD

**Issue by MaverickJones** [sparrowwallet/sparrow#820]
This is a pretty straightforward request, and I think can be accommodated.  It would be nice to be able to pick UTXOs from any arbitrary non-whirlpool account (or at least the first "n" accounts?) to be used in a Tx0 transaction for Whirlpool.

I use multiple accounts, some of which I've setup as multiple separate watch-only wallets on my mobile.  (It's not my use case, but I can imagine a setup where each account is permissioned differently, shared with different sets of watch-only users, etc.).  Currently, to add UTXOs from Accounts 1+, a pre-transaction needs to be done to move it to the Deposit Account #0.  This would let us avoid that first extra transaction (this seems prudent, given current blockspace debates taking place among the broader community of developers).

**Comment by craigraw** [comment]
I agree this would be useful, and has been in development from the Samourai team who maintain the Java Whirlpool client. I hope to add support for this soon.

**Comment by Giszmo** [comment]
Can this and other features be closed after https://github.com/sparrowwallet/sparrow/releases/tag/1.9.0 removed Whirlpool?
